pkgpanda/__init__.py

```diff
@@ -678,9 +712,22 @@ def symlink_all(src, dest):
 
         active_buildinfo_full = {}
 
+        dcos_service_configuration = self._get_dcos_configuration_template()
+
         # Building up the set of users
         sysusers = UserManagement(self.__manage_users, self.__add_users)
 
+        def _get_service_names(_dir):
+            service_files = list(map(os.path.basename,
+                                     glob.glob(_dir + "/dcos.target.wants/*.service") +
```

Comment:
We potentially have .socket, .target and other things in here. I'd really rather do everything in dcos.target.wants. I am never a fan of globbing. The `.target` ones we definitely want. This logic doesn't match the rest of the logic in this code that does "everything", I'd rather not fork them.

We also very much shouldn't be only looking for a specific small subset of roles / hard coding the list of roles here. When we're activating packages, we have the full exact list of roles, need to use that.